cli/hzn.go

```diff
@@ -658,6 +657,21 @@ Environment Variables:
 	utilVerifyPubKeyFile := utilVerifyCmd.Flag("public-key-file", msgPrinter.Sprintf("The path of public key file (that corresponds to the private key that was used to sign) to verify the signature of stdin.")).Short('K').Required().ExistingFile()
 	utilVerifySig := utilVerifyCmd.Flag("signature", msgPrinter.Sprintf("The supposed signature of stdin.")).Short('s').Required().String()
 
+	smCmd := app.Command("secretsmanager", msgPrinter.Sprintf("List and manage resources in the secrets manager. This CLI does not read and pull secret details; to do so, you must authenticate as an administrator and use the secrets manager CLI."))
+	smOrg := smCmd.Flag("org", msgPrinter.Sprintf("The Horizon organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
+	smUserPw := smCmd.Flag("user-pw", msgPrinter.Sprintf("Horizon Exchange credentials to query secrets manager resources. The default is HZN_EXCHANGE_USER_AUTH environment variable. If you don't prepend it with the user's org, it will automatically be prepended with the value of the HZN_ORG_ID environment variable.")).Short('u').PlaceHolder("USER:PW").String()
+	smSecretCmd := smCmd.Command("secret", msgPrinter.Sprintf("List and manage secrets in the secrets manager."))
+	smSecretListCmd := smSecretCmd.Command("list", msgPrinter.Sprintf("Display the names of the secrets in the secrets manager."))
+	smSecretListName := smSecretListCmd.Arg("secretName", msgPrinter.Sprintf("List just this one secret. Returns a boolean indicating the existence of the secret. This is the name of the secret used in the secrets manager.")).String()
+	smSecretAddCmd := smSecretCmd.Command("add", msgPrinter.Sprintf("Add a secret to the secrets manager."))
+	smSecretAddName := smSecretAddCmd.Arg("secretName", msgPrinter.Sprintf("The unique name to be used by the secrets manager.")).Required().String()
+	smSecretAddFile := smSecretAddCmd.Flag("secretFile", msgPrinter.Sprintf("Filepath to a file containing the secret details. Mutually exclusive with --secretDetail. Specify -f- to read from stdin.")).Short('f').String()
+	smSecretAddKey := smSecretAddCmd.Flag("secretKey", msgPrinter.Sprintf("A key for the secret.")).Short('k').Required().String()
+	smSecretAddDetail := smSecretAddCmd.Flag("secretDetail", msgPrinter.Sprintf("The secret details as a string. Mutually exclusive with --secretFile (-f).")).Short('d').String()
+	smSecretAddOverwrite := smSecretAddCmd.Flag("overwrite", msgPrinter.Sprintf("Overwrite the existing secret if it exists in the secrets manager. It will skip the 'do you want to overwrite' prompt.")).Short('O').Bool()
+	smSecretRemoveCmd := smSecretCmd.Command("remove", msgPrinter.Sprintf("Remove a secret in the secrets manager."))
```

Comment:
Please add "rm" as an alias for this command.

Reply:
addressed in recent changes